packages/libsqlite/build.sh

```diff
@@ -2,32 +2,31 @@ TERMUX_PKG_HOMEPAGE=https://www.sqlite.org
 TERMUX_PKG_DESCRIPTION="Library implementing a self-contained and transactional SQL database engine"
 TERMUX_PKG_LICENSE="Public Domain"
 TERMUX_PKG_MAINTAINER="@termux"
-# Note: Updating this version requires bumping libsqlite-tcl package as well.
-_SQLITE_MAJOR=3
-_SQLITE_MINOR=45
-_SQLITE_PATCH=2
+TERMUX_PKG_VERSION="3.45.3"
 _SQLITE_YEAR=2024
-TERMUX_PKG_VERSION=${_SQLITE_MAJOR}.${_SQLITE_MINOR}.${_SQLITE_PATCH}
-TERMUX_PKG_SRCURL=https://www.sqlite.org/${_SQLITE_YEAR}/sqlite-autoconf-${_SQLITE_MAJOR}${_SQLITE_MINOR}0${_SQLITE_PATCH}00.tar.gz
-TERMUX_PKG_SHA256=bc9067442eedf3dd39989b5c5cfbfff37ae66cc9c99274e0c3052dc4d4a8f6ae
+TERMUX_PKG_SRCURL=https://www.sqlite.org/${_SQLITE_YEAR}/sqlite-autoconf-$(sed 's/\./''/; s/\./0/' <<< "$TERMUX_PKG_VERSION")00.tar.gz
+TERMUX_PKG_SHA256=b2809ca53124c19c60f42bf627736eae011afdcc205bb48270a5ee9a38191531
+TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="zlib"
+TERMUX_PKG_BUILD_DEPENDS="tcl"
 TERMUX_PKG_BREAKS="libsqlite-dev"
 TERMUX_PKG_REPLACES="libsqlite-dev"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --enable-readline
 --enable-fts3
 "
 
-termux_step_post_get_source() {
-	# Version guard
-	local ver_s=${TERMUX_PKG_VERSION#*:}
-	local ver_t=$(. $TERMUX_SCRIPTDIR/packages/libsqlite-tcl/build.sh; echo ${TERMUX_PKG_VERSION#*:})
-	if [ "${ver_s}" != "${ver_t}" ]; then
-		termux_error_exit "Version mismatch between libsqlite and libsqlite-tcl."
-	fi
-}
-
 termux_step_pre_configure() {
 	CPPFLAGS+=" -Werror -DSQLITE_ENABLE_DBSTAT_VTAB=1 -DSQLITE_ENABLE_COLUMN_METADATA=1 -DSQLITE_ENABLE_UPDATE_DELETE_LIMIT=1"
 	LDFLAGS+=" -lm"
 }
+
+termux_step_post_make_install() {
```

Comment:
This looks to be the same idea used in `libncnn` and `qbittorrent`'s builds?
https://github.com/termux/termux-packages/blob/589d25b32d9e84f3c74da9e35373eea0db890dd8/x11-packages/qbittorrent/build.sh#L21-L27

Reply:
Correct. #20058